docs/DeveloperGuide.adoc

```diff
@@ -763,20 +848,197 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. iContacts shows an error message.
 +
 Use case resumes at step 2.
 
-{More to be added}
+[discrete]
+=== Use case: UC2 -  Export Contacts
+
+*MSS*
+
+1.  User selects contacts to be exported based on current list
+2.  User requests for creating file containing selected contacts
```

Comment:
use cases should capture the user-system interaction. in light of this, there should be something between 1 and 2 that the system does

Reply:
Agreed. Taken care of